AdblockPlusSafariExtension/AdblockPlus+Extension.m

```diff
@@ -59,7 +59,7 @@ - (NSURL *)activeFilterListURLWithWhitelistedWebsites
   NSURL *original = self.activeFilterListsURL;
   NSString *fileName = original.lastPathComponent;
 
-  if (fileName == nil || [fileName isEqual:@"empty.json"] || self.whitelistedWebsites.count == 0)  {
+  if (fileName == nil || [fileName isEqual:@"empty.json"])  {
```

Comment:
Filter lists must be processed every time, since they must be converted into safari compatible format.